lib/folio_sync/folio/client.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+class FolioSync::Folio::Client < FolioApiClient
+  def self.instance
+    @instance ||= self.new(
+      FolioApiClient::Configuration.new(
+        url: Rails.configuration.folio['base_url'],
+        username: Rails.configuration.folio['username'],
+        password: Rails.configuration.folio['password'],
+        tenant: Rails.configuration.folio['tenant'],
+        timeout: Rails.configuration.folio['timeout']
+      )
+    )
+    @instance
+  end
+
+  def check_health
+    self.get('/admin/health')
+  end
+
+  # @param hrid [String] The HRID (BIBID) of the instance record to fetch.
+  def get_marc_record(hrid)
+    # Returns Marc::Record
+    self.find_marc_record(instance_record_hrid: hrid)
+  end
+
+  def create_or_update_folio_record(bibid)
+    folio_marc = get_marc_record(bibid)
+
+    if folio_marc
```

Comment:
For this ticket, we're not preserving any data from FOLIO, so `folio_marc` is not used. 

Also, depending on how FOLIO data import works with new MARC records, we might modify this if/else.